uploadPatchSet.sh

```diff
@@ -3,37 +3,37 @@ TARGET="$1"
 ENVIRONMENT="$2"
 
 case $# in
-  2)
-    ENVIRONMENT="$2"
-    ;;
-  1)
-    ENVIRONMENT="local"
-    ;;
-  *)
-    echo "not enough arguments supplied.  You must supply the flowName to this command."
-    exit 1
-    ;;
-esac    
+    2)
+        ENVIRONMENT="$2"
+        ;;
+    1)
+        ENVIRONMENT="local"
+        ;;
+    *)
+        echo "not enough arguments supplied.  You must supply the flowName to this command."
+        exit 1
+        ;;
+esac
 
 source setEnvForUpload.sh $ENVIRONMENT
 
 if [ -z $FLOW_TOKEN ] ;
 then
-	exit 1
+	  exit 1
 else
-	http_response=$(curl $CURL_ARGS -s -o uploadFlowResponse.txt -w "%{http_code}" -X POST -H "flow-token: $FLOW_TOKEN" -H "Content-Type: application/json" "$HOST/repository/patchSets" --data-binary "@src/main/patchSets/$TARGET.json")
+	  http_response=$(curl $CURL_ARGS -s -o uploadFlowResponse.txt -w "%{http_code}" -X POST -H "flow-token: $FLOW_TOKEN" -H "Content-Type: application/json" "$HOST/repository/patchSets" --data-binary "@src/main/patchSets/$TARGET.json")
 fi
 
 if [ $http_response != "200" ];
 then
-  if [ $http_response == "302" ];
-  then
-    echo "Got unexpected HTTP response ${http_response}. This is likely due to your token being incorrect."
-  else
-    echo "Got unexpected HTTP response ${http_response}. This is likely an error."
-  fi
+    if [ $http_response == "302" ];
+    then
+        echo "Got unexpected HTTP response ${http_response}. This is likely due to your token being incorrect."
+    else
+        echo "Got unexpected HTTP response ${http_response}. This is likely an error."
+    fi
 else
-  cat uploadFlowResponse.txt
+    cat uploadFlowResponse.txt
```

Comment:
we might want to put this in the unexpected error space as well, so that we can see the errors